aten/src/ATen/native/Linear.cpp

```diff
@@ -16,6 +16,11 @@
 
 namespace at { namespace native {
 
+Tensor bias(const Tensor& input, const Tensor& bias) {
+  auto output = at::add(input, bias);
+  return output;
```

Comment:
Note: the + operator is overloaded for tensors, so input + bias works here as well.